tensorflow/compiler/xla/service/llvm_ir/llvm_util.cc

```diff
@@ -93,6 +93,20 @@ llvm::CallInst* EmitCallToIntrinsic(
   return b->CreateCall(intrinsic, AsArrayRef(operands));
 }
 
+llvm::CallInst* EmitCallToTargetIntrinsic(
+    TargetIntrinsicEnum intrinsic_en, absl::Span<llvm::Value* const> operands,
+    absl::Span<llvm::Type* const> overloaded_types,
+    LLVMTargetIRBuilder& llvm_target_ir_builder) {
```

Comment:
s/llvm_target_ir_builder/target_ir_builder/?  `llvm` is kind of redundant here, as we're already in namespace llvm, and "llvm" is already in the type name.